pyproject.toml

```
@@ -1,13 +1,80 @@
 [build-system]
-# Further build requirements - like numpy & cython - come from setup.py via
-# the PEP 517 interface.
 requires = [
-    "setuptools",
-    "wheel",
+    "Cython ~=0.29",
     "oldest-supported-numpy",
     "pkgconfig",
-    "Cython >=0.29; python_version<'3.8'",
-    "Cython >=0.29.14; python_version=='3.8'",
-    "Cython >=0.29.15; python_version>='3.9'",
+    "setuptools >=61",
 ]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "h5py"
+description = "Read and write HDF5 files from Python"
+authors = [
+    {name = "Andrew Collette", email = "andrew.collette@gmail.com"},
+]
+maintainers = [
+    {name = "Thomas Kluyver", email = "thomas@kluyver.me.uk"},
```

```suggestion
    {name = "Thomas Kluyver", email = "thomas@kluyver.me.uk"},
    {name = "Thomas A Caswell", email = "tcaswell@bnl.gov"},
```

in the address I would like to use for this.